ts/parts/LogarithmicAxis.ts

```diff
@@ -19,180 +26,267 @@ import H from './Globals.js';
-                this.getMinorTickInterval() :
-                options.tickInterval,
-            filteredTickIntervalOption = tickIntervalOption === 'auto' ?
-                null :
-                tickIntervalOption,
-            tickPixelIntervalOption =
-                (options.tickPixelInterval as any) / (minor ? 5 : 1),
-            totalPixelLength = minor ?
-                axisLength / axis.tickPositions.length :
-                axisLength;
-
-        interval = pick(
-            filteredTickIntervalOption,
-            axis._minorAutoInterval,
-            (realMax - realMin) *
-                tickPixelIntervalOption / (totalPixelLength || 1)
-        );
-
-        interval = normalizeTickInterval(
-            interval,
-            null as any,
-            getMagnitude(interval)
-        );
-
-        positions = axis.getLinearTickPositions(
-            interval,
-            realMin,
-            realMax
-        ).map(axis.log2lin);
+        // Third case: We are so deep in between whole logarithmic values that
+        // we might as well handle the tick positions like a linear axis. For
+        // example 1.01, 1.02, 1.03, 1.04.
+        } else {
+            var realMin = axis.lin2log(min),
+                realMax = axis.lin2log(max),
+                tickIntervalOption = minor ?
+                    axis.getMinorTickInterval() :
+                    options.tickInterval,
+                filteredTickIntervalOption = tickIntervalOption === 'auto' ?
+                    null :
+                    tickIntervalOption,
+                tickPixelIntervalOption =
+                    (options.tickPixelInterval as any) / (minor ? 5 : 1),
+                totalPixelLength = minor ?
+                    axisLength / axis.tickPositions.length :
+                    axisLength;
+
+            interval = pick(
+                filteredTickIntervalOption,
+                log.minorAutoInterval,
+                (realMax - realMin) *
+                    tickPixelIntervalOption / (totalPixelLength || 1)
+            );
+
+            interval = normalizeTickInterval(
+                interval,
+                void 0,
+                getMagnitude(interval)
+            );
+
+            positions = axis.getLinearTickPositions(
+                interval,
+                realMin,
+                realMax
+            ).map(axis.log2lin);
 
+            if (!minor) {
+                log.minorAutoInterval = interval / 5;
+            }
+        }
+
+        // Set the axis-level tickInterval variable
         if (!minor) {
-            axis._minorAutoInterval = interval / 5;
+            axis.tickInterval = interval;
         }
+        return positions;
     }
 
-    // Set the axis-level tickInterval variable
-    if (!minor) {
-        axis.tickInterval = interval;
+    public lin2log(num: number): number {
+        const axis = this.axis;
+        const lin2log = axis.options.linearToLogConverter;
```

Comment:
Remove the `linearToLogConverter`. It is not officially supported, plus without a reverse function, `logToLinearConverter`, it is unusable.